app/src/test/java/org/mozilla/fenix/perf/StartupTypeTelemetryTest.kt

```diff
@@ -35,7 +35,7 @@ private val validTelemetryLabels = run {
 
 private val activityClass = HomeActivity::class.java
 
-@RunWith(FenixRobolectricTestRunner::class)
+@RunWith(AndroidJUnit4::class)
```

Comment:
@MatthewTighe @MozillaNoah I'm trying to look into https://github.com/mozilla-mobile/fenix/issues/26381 https://github.com/mozilla-mobile/fenix/issues/26380 and the last change that took place to this class happened in this PR. Looking at the diff here, there is no context on why the test runner was changed or why it was part of this wallpaper patch in particular. Could you provide some context here?

Reply:
@jonalmeida Yeah, that is fully my fault. I started hitting the referenced error during CI and in a rush to try and land these wallpapers patches attempted a fix following some advice in [this comment chain](https://github.com/robolectric/robolectric/issues/5356). Clearly, it didn't actually resolve the issue and I recognize that this was a Not Cool Move on my part. I am happy to revert the change if we'd like